BuildDetails.md

```diff
@@ -0,0 +1,39 @@
+This is the build description guide that explains the build process for all FullScreenShenanigans projects. The guide will explain to you how the project is built and why it is built that way.
+
+### Table of Contents
+
+1. [Grunt Tasks](#grunt-tasks)
+
+## Grunt Tasks
+### clean
+The clean task cleans out the Distribution directory in the FullScreenShenanigans project. This occurs because if a new build has changes made, the Distribution folder must be recreated to reflect those changes.
+
+### tslint
+The tslint task make sure that the .ts files in the project are neat and readable. It checks each file for whitespaces and alignment issues and throws errors if it finds any.
```

Comment:
It does more than just whitespace and alignment issues. Naming conventions, unused variables, file endlines, missing semicolons, etc.